sdk/core/core-rest-pipeline/src/httpHeaders.ts

```diff
@@ -25,7 +25,7 @@ class HttpHeadersImpl implements HttpHeaders {
    * @param name - The name of the header to set. This value is case-insensitive.
    * @param value - The value of the header to set.
    */
-  public set(name: string, value: string | number): void {
+  public set(name: string, value: string | number | boolean): void {
```

Comment:
Don't we also need to update the function `createHttpHeaders` in this file to take the new shape as well?

Reply:
Yup, that one, and the constructor as well. Thanks for pointing that out

Reply:
Yes, pushed the change